<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainTable</class>
 <widget class="QMainWindow" name="MainTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>729</width>
    <height>513</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>729</width>
    <height>513</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>729</width>
    <height>513</height>
   </size>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="focusPolicy">
   <enum>Qt::WheelFocus</enum>
  </property>
  <property name="windowTitle">
   <string>GPA Calculator</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>321</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(52, 101, 164);
font: 57 23pt &quot;Century Schoolbook L&quot;;</string>
    </property>
    <property name="text">
     <string>Grade Point Average</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_btn">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(92, 53, 102);
font: 75 14pt &quot;Century Schoolbook L&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;-</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>691</width>
      <height>381</height>
     </rect>
    </property>
    <property name="cursor" stdset="0">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 57 14pt &quot;Century Schoolbook L&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>172</number>
    </attribute>
    <column>
     <property name="text">
      <string>Course Title</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Course Code</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Unit</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Grade</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="get_result">
    <property name="geometry">
     <rect>
      <x>573</x>
      <y>450</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(92, 53, 102);
font: 75 14pt &quot;Century Schoolbook L&quot;;</string>
    </property>
    <property name="text">
     <string>Get Result</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear_btn">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>450</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color:rgb(164, 0, 0);
font: 75 14pt &quot;Century Schoolbook L&quot;;</string>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
